ts-scripts/generateStaticTokens.ts

```diff
@@ -117,12 +117,17 @@ const formatEvmTokens = (tokens: PeggyTokenSource[]) =>
     tokenType: TokenType.Evm
   }))
 
-const formatErc20Tokens = (tokens: PeggyTokenSource[]) =>
-  tokens.map((token) => ({
-    ...token,
-    denom: `peggy${token.address}`,
-    tokenType: TokenType.Erc20
-  }))
+const formatErc20Tokens = (tokens: PeggyTokenSource[], network: Network) =>
+  tokens.map((token) => {
+    const denom = `peggy${token.address}`
+    const existingFactoryToken = getBankTokenMetadata(denom, network)
```

Comment:
We should retrieve the correct casing from supplyDenoms, peggy addresses doesn't exist on bankTokenMetadata